src/app/lib/config/services/serbian.js

```diff
@@ -26,7 +26,9 @@ const baseServiceConfig = {
   isTrustProjectParticipant: true,
   manifestPath: '/manifest.json',
   swPath: '/sw.js',
-  hasRadioSchedule: false,
+  radioSchedule: {
```

Comment:
Move this block after the `MostRead` the block for consistency.

Reply:
@EinsteinNjoroge  I do not think we need to do that here because this is a service with variants, so there are two `mostRead` blocks, one for each variant. We only need one `radioSchedule` block.